src/config/ci.ts

```diff
@@ -6,7 +6,7 @@ const config: ConfigInterface = {
   },
   modules: {
     latestVersion: '0.0.9',
-    oldestVersion: '0.0.5',
+    oldestVersion: '0.0.6',
```

Comment:
We forgot to update these in the `prepare` PR and my new change just made that fatal, so that's good. (Ideally we don't need to set these values explicitly at all, but...)